### ISSUE TITLE
Missing section - /disability/eligibility/hazardous-materials-exposure/agent-orange/

### ISSUE THREAD

**Issue by jhonnyoddball** [department-of-veterans-affairs/va.gov-team#15339]
## Issue Description
Section appears to be missing

## Example:
![Screen Shot 2020-10-27 at 12.14.14 PM.png](https://images.zenhubusercontent.com/5d89589c3ac3440001d19c89/be15e69f-ee89-4270-809e-3c7fe37cf6fc)

## Example File
`/disability/eligibility/hazardous-materials-exposure/agent-orange/index.html`

## Tasks
- [ ] Figure out why the discrepancy exists.
- [ ] Fix the issue

**Comment by jhonnyoddball** [comment]
Based on the [liquid template](https://github.com/department-of-veterans-affairs/vets-website/blob/09f8a9c6cbc511092c54d51600de0d8a857ac9b6/src/site/layouts/page.drupal.liquid#L33-L43), `fieldAlert` is required, however, it's return as `null` in the build process

```
{% if fieldAlert.length %}
   {% for alert in fieldAlert %}
      {% if alert.entity != empty %}
         {% include "src/site/blocks/alert.drupal.liquid" with alert = alert.entity %}
      {% endif %}
   {% endfor %}
{% else %}
   {% if fieldAlert.entity != empty %}
      {% include "src/site/blocks/alert.drupal.liquid" with alert = fieldAlert.entity %}
   {% endif %}
{% endif %}
```

Looking at `node.8ce9264e-b642-46eb-a51a-9ed94f9fe9f1.json`

```
    "field_alert": [
        {
            "target_type": "block_content",
            "target_uuid": "6ecdbf96-2a9e-4beb-9d95-d41fced1473b"
        }
    ],
```

And also at `block_content.6ecdbf96-2a9e-4beb-9d95-d41fced1473b.json`

```
    "status": [
        {
            "value": false
        }
    ],
```

While debugging the build process I found out that this is what it's overriding the node:

```
    // If the entity is unpublished
    if (!entity.status[0].value && !transformUnpublished) {
      return null;
    }
```

Basically, the paragraph is not showing due to the reference node is not published.

**Comment by kevwalsh** [comment]
I just confirmed that the alert block in question is in "draft" state in Drupal. 

I'll go ahead and move it to "published".  Seems like we have a bug in the graphql queries that hasn't been noticed until now.

**Comment by jhonnyoddball** [comment]
node was changed to `published`

Closing this tickets as I verified that the issue was solved
